WikiDrips/WikiTableViewController.swift

```diff
@@ -140,12 +153,23 @@ extension WikiTableViewController: UISearchResultsUpdating {
 // MARK: - UITableViewDataSourcePrefetching
 extension WikiTableViewController: UITableViewDataSourcePrefetching {
     func tableView(_ tableView: UITableView, prefetchRowsAt indexPaths: [IndexPath]) {
-        print("prefetchRowsAt \(indexPaths)")
-        indexPaths.forEach { generateImage(forItemAtIndex: $0) }
+        indexPaths.forEach {
+            checkImage(forItemAtIndex: $0)
+        }
     }
     
-    func tableView(_ tableView: UITableView, cancelPrefetchingForRowsAt indexPaths: [IndexPath]) {
-        print("cancelPrefetchingForRowsAt \(indexPaths)")
-        indexPaths.forEach { cancelImage(forItemAtIndex: $0) }
+// No need for cancelPrefetchingForRowsAt since changing search cancels pending operations and prefetchRowsAt populates a cache
+}
+
+// MARK: - ImageCache
+class ImageCache: NSCache<AnyObject, AnyObject> {
```

Comment:
We can probably change this to `NSCache<NSString, AnyObject>` to be stronger typed.